frontend/src/queries/nodes/InsightViz/InsightContainer.tsx

```diff
@@ -70,24 +70,8 @@ export function InsightContainer({
     disableLastComputation?: boolean
     insightMode?: ItemMode
 }): JSX.Element {
-    const {
```

Comment:
Sorry for leaving the commented out code here - I know it's not beautiful, but I'm using it as reminder for what still needs to be implemented with data exploration. Would be grateful if we can leave it in meanwhile.

Reply:
Ah ok I see - in general I'm for keeping the codebase clean and having other mechanisms for reminders but happy to revert these changes!

Reply:
done